htdocs/ticket/list.php

```diff
@@ -390,7 +390,7 @@
 		}
 		continue;
 	}
-	$mode_search = (($object->isInt($object->fields[$key]) || $object->isFloat($object->fields[$key])) ? 1 : 0);
+	$mode_search = ((array_key_exists($key, $object->fields) && $object->isInt($object->fields[$key]) || $object->isFloat($object->fields[$key])) ? 1 : 0);
```

Comment:
If first test is false because array_key_exists($key, $object->fields) is false, then the || is tested. But here we will get a warning because array_key_exists($key, $object->fields) is false.
Parenthesis should be around the || and not the &&